web_portal_link/views/views.xml

```diff
@@ -8,9 +8,9 @@
     <menuitem
         id="my_portal_view"
         name="My Page"
-        web_icon="my_portal_page,static/description/icon.png"
+        web_icon="web_portal_link,static/description/icon.png"
```

Comment:
@AungKoKoLin1997 
If you change module name, you have to chenge the code including module name

Reply:
@kanda999 
Sorry. I didn't notice that app icon. Thanks for pointing it out.